.gitignore

```diff
@@ -336,3 +336,4 @@ ASALocalRun/
 
 # Ignore Mac DS_Store files
 .DS_Store
+/src/Native/out/build/x64-Debug
```

Comment:
mm not sure why you had these files here, but I don't think we want to change the .gitignore as nothing should get dropped in this specific folder. Try removing this, and running a `git clean -fdx` from the root of the repo in order to delete all the files that are not checked in. Then run a `build.cmd` and you should see that nothing gets dropped there.

Reply:
It seems that CMake is being run when visual studio opens the folder. This is then creating the files. in /src/Native/out/build-X64-Debug. I'll investigate further.

Reply:
I have a bit more clarity now. This problem appears to have been introduced with Visual Studio 16.1 (It seems ok with the initial version of VS2019 and the latest VS2017). The CMake command line is as below.. the  -DCMAKE_INSTALL_PREFIX:PATH seems to be pointing into the Native\out directory whereas in earlier versions of VS then it is a temporary directory. I will try and see if this is intentional or otherwise....

 "C:\PROGRAM FILES (X86)\MICROSOFT VISUAL STUDIO\2019\ENTERPRISE\COMMON7\IDE\COMMONEXTENSIONS\MICROSOFT\CMAKE\CMake\bin\cmake.exe"  -G "Ninja" -DCMAKE_INSTALL_PREFIX:PATH="d:\source\dotnet\iot\src\native\out\install\x64-Debug" -DCMAKE_CXX_COMPILER:FILEPATH="C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.21.27702/bin/HostX64/x64/cl.exe" -DCMAKE_C_COMPILER:FILEPATH="C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.21.27702/bin/HostX64/x64/cl.exe"  -DCMAKE_BUILD_TYPE="Debug" -DCMAKE_MAKE_PROGRAM="C:\PROGRAM FILES (X86)\MICROSOFT VISUAL STUDIO\2019\ENTERPRISE\COMMON7\IDE\COMMONEXTENSIONS\MICROSOFT\CMAKE\Ninja\ninja.exe" "D:\source\dotnet\iot\src\Native"